src/screens/sync/index.tsx

```diff
@@ -25,7 +26,7 @@ const styles = StyleSheet.create({
   },
 });
 
-export const SyncScreen: SyncScreenComponent<"SyncMain"> = () => {
+export const SyncScreen: ScreenComponent<"Sync"> = ({ route }) => {
```

Comment:
unused variable `route` but leaving to make it easier to know that it's usable for upcoming work :)